cocos/core/scheduler.ts

```diff
@@ -35,19 +35,27 @@ const MAX_POOL_SIZE = 20;
 const idGenerator = new IDGenerator('Scheduler');
 
 export interface ISchedulable {
-    id?:string;
-    uuid?:string;
+    id?: string;
+    uuid?: string;
 }
 
 // data structures
 /**
  * @en A list double-linked list used for "updates with priority"
  * @zh 用于“优先更新”的列表
  * @class ListEntry
- * @param target not retained (retained by hashUpdateEntry)
+ * @param target
+ * @en target object, which is ISchedulable type, retained by hashUpdateEntry
+ * @zh 目标对象, 为ISchedulable类型. 被hashUpdateEntry持有
  * @param priority
+ * @en the priority
+ * @zh 优先级
  * @param paused
- * @param markedForDeletion selector will no longer be called and entry will be removed at end of the next tick
+ * @en whether is paused
+ * @zh 是否被暂停
+ * @param markedForDeletion
+ * @en mark for deletion. if true, selector will no longer be called and entry will be removed at end of the next tick
+ * @zh 删除标记, 当为true时, selector 将不再被调用，并且entry将在下一个tick结束时被删除
  */

```

Comment:
I think parameters should be described in constructor, not in class description.

Reply:
got it.